components/typeform/actions/update-dropdown-multiple-choice-ranking/update-dropdown-multiple-choice-ranking.mjs

```diff
@@ -112,52 +108,44 @@ export default {
         }),
       ],
     },
-    choices: {
-      type: "string[]",
-      label: "Field's choices",
-      description: "Please select the choices you want to include in your Question. In case you want to build yourself the selected options, the structure should look like this `{{[{\"label\":\"Choice 1\"}]}}`. Please provide at least one option.",
-      options() {
-        const { properties } = JSON.parse(this.field);
-        return properties.choices.map((choice) => ({
-          label: choice.label,
-          value: JSON.stringify(choice),
-        }));
-      },
+    choice: {
+      type: "string",
+      label: "New choice",
+      description: "Please add the new choice you want to include in your Question",
```

Comment:
The **Update Dropdown, Multiple Choice or Ranking Question Options** action on Zapier seems to replace existing options/choices for a question with a list of new choices.
<img width="300" alt="zapier-multiple-choices" src="https://user-images.githubusercontent.com/19861096/139770627-19c4d3aa-2ac6-4880-ba6b-b9a142621ae8.png">
Do you think we should replace the choices for a question instead of adding a choice with this action?

Reply:
actually I've been discussing about this one with @dannyroosevelt can you check the chat [here](https://pipedream-users.slack.com/archives/C01E5KCTR16/p1635446923020500?thread_ts=1635446184.019000&cid=C01E5KCTR16) please?